logilica_weekly_report/page_dashboard.py

```diff
@@ -41,10 +37,9 @@ def download_team_dashboards(
                     dashboard,
                     team,
                 )
-                NavigationPanel(self.page).navigate(
-                    link_name=dashboard, menu_dropdown=menu_dropdown
-                )
-                self.download_dashboard_to(path=base_dir_path / options["Filename"])
+                self.page.goto(url=options["url"])
```

Comment:
Would there be a value to keep both navigation options, .g. to get to the dashboard by a direct url and by navigation panel?

Reply:
Yes and no:  obviously, the ability to manipulate the filters is very powerful (and I worked very hard to create it!), but it strikes me as potentially fragile (and, it wasn't actually that complicated, once it was done), so I'm not inclined to try to maintain it (especially if nothing currently needs or uses it).

On the other hand, using the saved URL is very clean, simple, and, hopefully, reliable.  (That was a superb suggestion!!  Thank you!)  So, I'd prefer to move forward with (just) that, until some other need arises.

Reply:
Makes sense.